tests/unit/maxmemory.tcl

```diff
@@ -493,3 +493,57 @@ start_server {tags {"maxmemory external:skip"}} {
         if {$::verbose} { puts "evicted: $evicted" }
     }
 }
+
+
+start_server {} {
+    set replica [srv 0 client]
+    set replica_host [srv 0 host]
+    set replica_port [srv 0 port]
+    start_server {} {
+        set master [srv 0 client]
+        set master_host [srv 0 host]
+        set master_port [srv 0 port]
+
+        # Start the replication process...
+        $replica replicaof $master_host $master_port
+        wait_for_sync $replica
+        after 1000
```

Comment:
why do we need a replica?
`attach_to_replication_stream` will work without one.

also, why `after 1000`? (did you copy it from somewhere?)
p.s. don't we need to set the ping period to a very high value to make sure one won't make it's way into our stream and fail the test?

Reply:
right about both, deleted the replica